framework/widgets/ActiveField.php

```diff
@@ -315,10 +317,17 @@ public function error($options = [])
      *
      * @return $this the field object itself
      */
-    public function hint($content, $options = [])
+    public function hint($content = null, $options = [])
```

Comment:
@PowerGamer1 It will break any class that override `hint()` method using old signature: https://3v4l.org/IoSqL